config/production.json

```diff
@@ -69,6 +69,7 @@
 		"plans/personal-plan": true,
 		"post-editor/author-selector": true,
 		"post-editor/insert-menu": true,
+		"post-editor/track-paste-events": true,
```

Comment:
Gave it a second thought: if we need that, we can add it later. Remove flags for now.